src/index.js

```diff
@@ -22,7 +22,8 @@ const CLASS_PICKER = 'emoji-picker';
 const CLASS_PICKER_CONTENT = 'emoji-picker__content';
 
 const DEFAULT_OPTIONS = {
-  position: 'right-start'
+  position: 'right-start',
+  rootElement: 'body'
```

Comment:
Instead of the string `body`, would it work to just use `document.body` as the default? This way you wouldn't need the `if` checks, you could just have `rootElement.appendChild(this.pickerEl)` on its own.

Reply:
yeah, will do 👍